src/main/kotlin/com/example/plugins/Routing.kt

```diff
@@ -6,27 +6,53 @@ import io.ktor.server.response.*
 
 fun Application.configureRouting() {
     val poolInfoController: PoolInfoController = PoolInfoController()
+    val simpleModbusRTURelay:SimpleModbusRTURelay = SimpleModbusRTURelay();
     routing {
         get("/pool-info"){
             return@get call.respondText(poolInfoController.answerServer()!!, status = HttpStatusCode.OK)
         }
+        post("/test"){         /// testing post
+            println("test work")
+            return@post call.respondText("Test work rezalt - " +  simpleModbusRTURelay.onRelay(1) , status = HttpStatusCode.OK)
+        }
+        post ("/relay/0/1"){
```

Comment:
Please use path pattern to avoid code duplication.
https://ktor.io/docs/routing-in-ktor.html#define_route